metered-channel/src/unbounded.rs

```diff
@@ -153,14 +158,16 @@ impl<T> UnboundedMeteredSender<T> {
 	}
 
 	/// Attempt to send message or fail immediately.
-	pub fn unbounded_send(
-		&self,
-		msg: T,
-	) -> result::Result<(), mpsc::TrySendError<MaybeTimeOfFlight<T>>> {
+	pub fn unbounded_send(&self, msg: T) -> result::Result<(), TrySendError<MaybeTimeOfFlight<T>>> {
 		let msg = self.prepare_with_tof(msg);
-		self.inner.unbounded_send(msg).map_err(|e| {
+		self.inner.try_send(msg).map_err(|e| {
 			self.meter.retract_sent();
 			e
 		})
 	}
+
+	/// Returns the current number of messages in the channel
+	pub fn len(&self) -> usize {
+		self.inner.len()
+	}
```

Comment:
Not sure we want to expose this publicly, it can create a false sense of introspectability

Reply:
I was thinking about using `len` in case of timeouts/errors. But why `false sense` - it really shows the number of messages being active in the channel which might be useful for some cases (e.g. if we want to send some `ok to drop` message or not).

Reply:
I didn't think of that case, but the accuracy here might be off too and not reflect the state at the time of error.

The whole ok to drop logic should eventually live in the channel as i.e. part of a closure which could access whatever info is provided. At least that be my favored approach since one can control the context of such closure.